crowbar_framework/app/models/service_object.rb

```diff
@@ -1459,6 +1486,11 @@ def apply_role(role, inst, in_queue)
     restore_to_ready(applying_nodes)
     process_queue unless in_queue
     [200, {}]
+  rescue StandardError => e
+    @logger.fatal("apply_role: Uncaught exception #{e.message} #{e.backtrace.join("\n")}")
+    message = "Failed to apply the proposal: uncaught exception (#{e.message})"
+    update_proposal_status(inst, "failed", message)
+    [405, message]
```

Comment:
I don't believe this should be in the backport.

It's not wrong, but this was not in any of the cherry-picked commits.